### PR TITLE
LPD-29294, LPD-29407 Add option to show the popover warning after hiding it

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPD-28182](https://liferay.atlassian.net/browse/LPD-28182)

image

image

https://github.com/user-attachments/assets/60e5aa00-a6d6-484b-874a-e7ae3c540cac

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
The failures are not related, same tests have been failing for a while: https://testray.liferay.com/#/project/35392/routines/1816788/build/15398668

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+		return HashMapBuilder.<String, Object>put(
+			"action", getNamespace(portletRequest) + "CTNotifications"
+		).put(
+			"globalAction", true
+		).build();
+	}
+
+	@Override
+	public String getJspPath() {
+		return "/publications/view_ct_notifications.jsp";
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java:80]: /publications/configuration/icon/ct_notfications.jsp

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	public Map<String, Object> getContext(PortletRequest portletRequest) {
+		String redirect = PortletURLBuilder.create(
+			PortletURLFactoryUtil.create(
+				portletRequest, CTPortletKeys.PUBLICATIONS,
+				PortletRequest.RENDER_PHASE)
+		).setMVCRenderCommandName(
+			"/change_tracking/view_publications"
+		).buildString();
+
+		portletRequest.setAttribute("redirect", redirect);
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java:54]: Not needed

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+			WebKeys.THEME_DISPLAY);
+
+		ResourceURL saveDisplayPreferenceURL =
+			(ResourceURL)_portal.getControlPanelPortletURL(
+				portletRequest, themeDisplay.getScopeGroup(),
+				CTPortletKeys.PUBLICATIONS, 0, 0,
+				PortletRequest.RESOURCE_PHASE);
+
+		saveDisplayPreferenceURL.setResourceID(
+			"/change_tracking/save_display_preference");
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/configuration/icon/CTNotificationsPortletConfigurationIcon.java:66]: Use PortletURLBuilder

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
ci:forward:force

**Pull request closed without merging**